data.sql

```diff
@@ -70,4 +70,62 @@ WHERE animals.name = 'Charmander' AND owners.full_name = 'Melody Pond'
 UPDATE animals
 SET owner_id = owners.id FROM owners
 WHERE animals.name = 'Angemon' AND owners.full_name = 'Dean Winchester'
-  OR animals.name = 'Boarmon' AND owners.full_name = 'Dean Winchester';
\ No newline at end of file
+  OR animals.name = 'Boarmon' AND owners.full_name = 'Dean Winchester';
+
+  INSERT INTO vets (name, age, date_of_graduation) VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations VALUES
+  ((SELECT id FROM species WHERE name = 'Pokemon'),
+    (SELECT id FROM vets WHERE name = 'William Tatcher')),
+  ((SELECT id FROM species WHERE name = 'Digimon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+  ((SELECT id FROM species WHERE name = 'Pokemon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+  ((SELECT id FROM species WHERE name = 'Digimon'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'));
```

Comment:
- [ ] Let us use this way again to insert the data into specializations

INSERT INTO specializations (species_id, vet_id)

 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));